### ISSUE TITLE
Should Getting Started be a markdown file instead?

### ISSUE THREAD

**Issue by jina** [bradfrost/style-guide-guide#19]
It currently has the `html` extension but is using markdown content.

https://github.com/bradfrost/style-guide-guide/blob/master/getting-started.html

**Comment by jina** [comment]
Same with gallery.html

**Comment by bradfrost** [comment]
@jina Good suggestion! These are `html` files because they're stripped down versions of client projects, which ultimately had quite a bit of HTML content inside of them. I'm not sure if it's worth switching over to `.md` extensions or not, but I suppose it could go either way.

**Comment by jina** [comment]
@bradfrost ah that wasn't really a suggestion. Was more of a potential bug report. But maybe jekyll can parse markdown on non-markdown files? :)

**Comment by bradfrost** [comment]
> But maybe jekyll can parse markdown on non-markdown files?

@jina yep, they can! And I suppose the opposite is true too (markdown files can parse HTML). I'm going to leave this as-is for now, mostly because I'm lazy and it's all working correctly. If you want to submit a PR, go for it and I'll buy you a drink! :)

**Comment by Ryuno-Ki** [comment]
According to the Markdown „spec” at https://daringfireball.net/projects/markdown/syntax#html HTML in Markdown is totally valid.

**Comment by jina** [comment]
I know that. :) I wasn't talking about HTML in markdown. I was talking about markdown rendering properly in a file that has an HTML file extension.

**Comment by Ryuno-Ki** [comment]
Like [reveal.js does](https://github.com/hakimel/reveal.js#markdown)?

They are using :octocat: [marked](/chjj/marked)
